### ISSUE TITLE
Create table with all approaches using deep learning for localization tasks

### ISSUE THREAD

**Issue by DanielCoelho112** [DanielCoelho112/synfeal#38]
Build a table that contains the main ideas of the approaches, such as:

- input data
- output data
- loss function
- results obtained
- key characteristics of the neural network

**Comment by DanielCoelho112** [comment]
The table is in this paper: https://arxiv.org/abs/1704.00390
